google-cloud-spanner/src/test/java/com/google/cloud/spanner/MockSpannerServiceImpl.java

```diff
@@ -1872,7 +1872,16 @@ private Transaction getTemporaryTransactionOrNull(TransactionSelector tx) {
   @Override
   public void beginTransaction(
       BeginTransactionRequest request, StreamObserver<Transaction> responseObserver) {
-    requests.add(request);
+    // Skip storing the explicit BeginTransactionRequest used to verify read-write transaction
```

Comment:
nit: add a TODO here so it is easier to find

Reply:
Done